packages/connectivity/connectivity_platform_interface/lib/src/utils.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:connectivity_platform_interface/connectivity_platform_interface.dart';
 
 /// Convert a String to a ConnectivityResult value.
-ConnectivityResult parseConnectivityResult(String state) {
+ConnectivityResult parseConnectivityResult(String? state) {
```

Comment:
looks like `state` could be non-null, right?

Reply:
done